bl-user-setup

```diff
@@ -18,15 +18,15 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # USER="$1" USER is exported by lightdm
```

Comment:
How would the system handle *user@domain1* and *user@domain2*? Two different people perhaps.